src/helpers/amoc_xmpp_mam.erl

```diff
@@ -0,0 +1,96 @@
+-module(amoc_xmpp_mam).
+
+-include_lib("exml/include/exml.hrl").
+
+-define(V(X), (fun amoc_config_validation:X/1)).
+
+-required_variable(
```

Comment:
I get your point, let's raise this with @DenysGonchar when he's back.

Reply:
https://esl.github.io/amoc/doc/configuration.html

```
NB: the reason why the -required_variable(...) is preferred over the usual behaviour callback is because the orchestration tools can easily extract the attributes even without the compilation, while configuring via a callback, requires a successful compilation of the module. As an example, a module:
```

Attribute is used to allow tide analyze scenarios w/o compilation. But as we're moving away from tide, we can change it. Let's discuss it later.